test/integration/branches.test.ts

```diff
@@ -1,34 +1,50 @@
 import { execSync } from 'child_process';
-import realFetch from 'cross-fetch';
-import dotenv from 'dotenv';
-import { join } from 'path';
-import { afterAll, beforeAll, describe, expect, test, vi } from 'vitest';
+import { afterAll, afterEach, beforeAll, beforeEach, describe, expect, test } from 'vitest';
 import { XataApiClient } from '../../packages/client/src';
 import { getCurrentBranchName } from '../../packages/client/src/util/config';
 import { XataClient } from '../../packages/codegen/example/xata';
+import { setUpTestEnvironment, TestEnvironmentResult } from '../utils/setup';
+
+let api: XataApiClient;
+let workspace: string;
+let database: string;
+let hooks: TestEnvironmentResult['hooks'];
+let fetch: TestEnvironmentResult['clientOptions']['fetch'];
+
+let getBranchOptions: {
```

Comment:
Nit (non-blocking): I might be misunderstanding. But when I was reading this code I thought this was a function because it starts with get.